### DISCUSSION TITLE
Is raylib only support .m3d and .iqm model animation?

### DISCUSSION THREAD

**Discussion by wlxmhls** [raysan5/raylib#2803] [Q&A] [answered]
Hello,

I want to play model animations in raylib. In the example code, I find an .iqm model animations sample. And in the rmodels.c, I find two animation loading functions:
```
static ModelAnimation *LoadModelAnimationsIQM(const char *fileName, unsigned int *animCount);    // Load IQM animation data
static ModelAnimation *LoadModelAnimationsM3D(const char *fileName, unsigned int *animCount);    // Load M3D animation data
```

I want to know if raylib support play .gltf animations. I make models using Blender, and the default export formats do not include .iqm or .m3d, but do .gltf.

Thanks!

**Comment by raysan5** [comment, marked as answer]
@wlxmhls raylib does not support gltf animations at the moment.

**Comment by Rayyy369** [comment]
I'll start building ASAP. I'm eager to explore this world your created. Few
things came up

**Reply by wlxmhls** [reply]
Thanks for your work
